fili-core/src/main/java/com/yahoo/bard/webservice/web/endpoints/TablesServlet.java

```diff
@@ -256,6 +260,70 @@ public Response getTableByGrain(
         }
     }
 
+
+    @GET
+    @Timed
+    @Produces(MediaType.APPLICATION_JSON)
+    @Path("/{tableName}/{granularity}/{dimensions:.*}")
+    public Response getTableAvailability(
+            @PathParam("tableName") String tableName,
+            @PathParam("granularity") String granularity,
+            @PathParam("dimensions") List<PathSegment> dimensions,
+            @QueryParam("metrics") String metrics,
+            @QueryParam("dateTime") String intervals,
+            @QueryParam("filters") String filters,
+            @Context UriInfo uriInfo,
+            @Context final ContainerRequestContext containerRequestContext
+    ) {
+        RequestLog.startTiming(this);
```

Comment:
Everything should be under the `try` block. 
The constructor for `TablesApiRequest` can throw `BadApiRequestException` which would cause us to never close the timer. Related to #362